### PR TITLE
Stop registering detailed guide root paths

### PR DESCRIPTION
**Pull request by binaryberry**
This is part of our [plan](https://docs.google.com/a/digital.cabinet-office.gov.uk/spreadsheets/d/1xMgOH3ha7dzoPaHJw9VAu9hlCvAm3ifwbBABbuFNMDQ/edit?usp=sharing) to move detailed guides under guidance/.

cc @boffbowsh @benilovj

### CODE REVIEW

**Comment by boffbowsh** [comment]
:+1:

File: app/models/registerable_edition.rb [hunk 1]
```diff
@@ -89,8 +89,6 @@ def published?
   end
 
   def detailed_guide_paths
-    old_slug = slug.sub(%r{^guidance/}, '')
-    ["/#{old_slug}"] + edition.non_english_translations.map { |t| "/#{edition.slug}.#{t.locale}" } +
-      ["/#{slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
+    ["/#{slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
```
Comment (edds) [inline app/models/registerable_edition.rb]: This is slightly odd to read as it uses two different slug methods which produce different kinds of slug. Would it be better as either 

```ruby
["/#{slug}"] + edition.non_english_translations.map { |t| "/#{slug}.#{t.locale}" }
```

or 

```ruby
["/guidance/#{edition.slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
```

Which I think (and could be wrong) is the intent behind the line.

Comment (boffbowsh) [inline app/models/registerable_edition.rb]: Yes, you're right. I think your first suggestion works best.

Comment (binaryberry) [inline app/models/registerable_edition.rb]: Yes, good spot. Amended now!

**Pull request merged**